website/docs/r/nsxv_dhcp_relay.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxv_dhcp_relay"
+sidebar_current: "docs-vcd-resource-nsxv-dhcp-relay"
+description: |-
+  Provides an NSX edge gateway DHCP relay configuration resource.
+---
+
+# vcd\_nsxv\_dhcp\_relay
+
+Provides a vCloud Director Edge Gateway DHCP relay configuration resource. The DHCP relay capability
+provided by NSX in vCloud Director environment allows to leverage existing DHCP infrastructure from
+within vCloud Director environment without any interruption to the IP address management in existing
+DHCP infrastructure. DHCP messages are relayed from virtual machines to the designated DHCP servers
+in your physical DHCP infrastructure, which allows IP addresses controlled by the NSX software to
+continue to be in synch with IP addresses in the rest of your DHCP-controlled environments. 
+
+Supported in provider *v2.6+*
+
+## Example Usage 1 (Minimal configuration)
+
+```hcl
+resource "vcd_nsxv_dhcp_relay" "relay_config" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+
+  ip_addresses = ["1.1.1.1"]
+
+  relay_agent {
+    org_network = vcd_network_routed.test-routed[0].name
+  }
+}
+```
+
+## Example Usage 2 (Example of configuration with multiple relay agents)
+
+```hcl
+resource "vcd_nsxv_dhcp_relay" "relay_config" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+
+  ip_addresses = ["1.1.1.1", "2.2.2.2"]
+  domain_names = ["servergroups.domainname.com", "other.domain.com"]
+  ip_sets      = [vcd_ipset.myset1.name, vcd_ipset.myset2.name]
```

Comment:
Thinking that it may be clearer for new users if we defined these IP set resources in this example as well.

Reply:
Added